pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/AggregatedSubscriptionStats.java

```diff
@@ -28,6 +28,8 @@ public class AggregatedSubscriptionStats {
 
     public long msgBacklogNoDelayed;
 
+    public long backlogSize;
```

Comment:
Maybe we can use `msgBacklogSize` to match with the `msgBacklog`, and add it below with `msgBacklog`.

Reply:
Thanks for advice, addressed.